src/Pages/Main/Components/Slider/Slider.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import { nativeTouchData } from 'react-dom/test-utils';
+import { left, right } from '../../../../config';
+import SlideContents from '../SliderContents/SliderContents';
+import './Slider.scss';
+export default class Slider extends Component {
```

Comment:
```suggestion
import React, { Component } from 'react';
import SlideContents from '../SliderContents/SliderContents';
import { left, right } from '../../../../config';
import './Slider.scss';

export default class Slider extends Component {
```
- 잘못 import 된 값이 있습니다.
- import 순서 고쳐주세요!